docker-for-mac/multi-arch.md

```diff
@@ -54,3 +54,162 @@ Notice that this time, the `uname -a` output shows `armv7l` and
 
 Multi-architecture support makes it easy to build <a href="https://blog.docker.com/2017/11/multi-arch-all-the-things/" target="_blank">multi-architecture Docker images</a> or experiment with ARM images and binaries from your Mac.
 
+## Buildx Tech Preview
```

Comment:
@thaJeztah @adamparco - Should we rename this section to Buildx (Experimental) instead of Buildx Tech Preview?

Reply:
I have seen discussion on the proper naming.  Not sure what the final decision was.

Reply:
This was based on the discussions and the suggested text around 'Experimental feature' by Kent in the #amberjack-leads channel.

Reply:
Yes, I think it'll be "experimental"